pkg/sdkserver/sdkserver.go

```diff
@@ -246,19 +247,8 @@ func (s *SDKServer) Run(ctx context.Context) error {
 		return err
 	}
 
-	logLevel := agonesv1.SdkServerLogLevelInfo
-	// grab configuration details
-	if gs.Spec.SdkServer.LogLevel != "" {
-		logLevel = gs.Spec.SdkServer.LogLevel
-	}
-	s.logger.WithField("logLevel", logLevel).Debug("Setting LogLevel configuration")
-	level, err := logrus.ParseLevel(strings.ToLower(string(logLevel)))
-	if err == nil {
-		s.logger.Logger.SetLevel(level)
-	} else {
-		s.logger.WithError(err).Warn("Specified wrong Logging.SdkServer. Setting default loglevel - Info")
-		s.logger.Logger.SetLevel(logrus.InfoLevel)
-	}
+	currentLogLevel := s.logger.Logger.GetLevel()
+	s.logger.WithField("logLevel", currentLogLevel).Debug("Setting LogLevel configuration")
```

Comment:
```suggestion
```

Don't think we need these lines anymore.

Reply:
I've taken out these two lines.